core/src/test/java/nl/nn/adapterframework/util/ClassUtilsTest.java

```diff
@@ -324,6 +324,7 @@ private ClassLoader getBytesClassLoader() throws IOException, ConfigurationExcep
 
 		JarFileClassLoader cl = new JarFileClassLoader(localClassLoader);
 		cl.setJar(file.getFile());
+		cl.setBasePath(".");
```

Comment:
Zou dat basepath niet default op . moeten staan? Of is het alleen maar voor de test?

Reply:
Dit is alleen voor de test, default is het de naam van de configuratie die gebruik maakt van de betreffende classloader. omdat er geen configuratie is in de test zorgt dit er voor dat het basepath genegeerd wordt.